### ISSUE TITLE
[Bug][Dropdowns] Axe errors

### ISSUE THREAD

**Issue by iadnonov** [telerik/kendo-react#1985]
## I'm submitting a...
* Bug report 

## Current behavior

* MultiSelect component - Missing `aria-label`
* ComboBox and DropDownList components do not set `aria-controls` when components are open

## Expected behavior

All the components should not valuate axe errors

## Minimal reproduction of the problem with instructions
MultiSelect component:
  1. Open the following demo: https://www.telerik.com/kendo-react-ui/components/dropdowns/multiselect/
  2. Test the component with axe DevTools
  
ComboBox component: 
  1. Open the following demo: https://stackblitz.com/edit/react-mltuam-yz5pqg?file=app%2Fmain.jsx
  2. Test the component with axe DevTools

Ticket ID: 1640852

**Comment by silviyaboteva** [comment]
This pull request is included in version 8.3.0-develop.1.
